src/test/java/com/celements/model/classes/fields/ListFieldTest.java

```diff
@@ -40,9 +41,8 @@
   @Before
   public void setUp() throws Exception {
     super.setUp();
```

Comment:
please do not override the default setUp method. Instead add a different named method with Before annotation.